volumes_test.go

```diff
@@ -9,14 +9,14 @@ import (
 func waitVolumeActive(id string, c *Client) (*Volume, error) {
 	// 15 minutes = 180 * 5sec-retry
 	for i := 0; i < 180; i++ {
-		<-time.After(5 * time.Second)
 		c, _, err := c.Volumes.Get(id)
 		if err != nil {
 			return nil, err
 		}
 		if c.State == "active" {
 			return c, nil
 		}
+		<-time.After(5 * time.Second)
```

Comment:
Did just a tiny adjustment here to save time when waiting for volumes in a loop (made the longer test case from 90s to 45s). : )